packages/plugin-tickets-api/src/models/definitions/boards.ts

```
@@ -97,6 +97,7 @@ export interface IPipeline extends ICommonFields {
   isCheckDate?: boolean;
   isCheckUser?: boolean;
   isCheckDepartment?: boolean;
+  isCheckBranch?: Boolean;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)